src/Core/Scripting/OOJSStation.m

```diff
@@ -947,31 +947,38 @@ static JSBool StationAddShipToShipyard(JSContext *context, uintN argc, jsval *vp
 		return NO;
 	}
 
+	// make sure the station has a shipyard
+	if ([station hasShipyard] == false) {
```

Comment:
Rather than `false`, probably better to use the ObjC `NO` here. But personally, I would much much prefer the form
`if (![station hasShipyard])`
which is what is used in about 90% of the codebase.